src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.html

```diff
@@ -46,8 +46,10 @@
                         </template>
 
                         <template if:true={isLightningCombobox}>
-                            <c-ge-form-field-picklist field-name={targetFieldApiName}
+                            <c-ge-form-field-picklist class-name="slds-m-around_none"
```

Comment:
Are you starting to include the attributes to support automation that you worked on with Sravani? This comment not actually related to this line)

Reply:
Not yet in this case. Sravani is whipping up a work item for that and running it by Ali I believe.